defaults/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+consul_version: 1.0.6
+
+consul_ip: "{{ ansible_default_ipv4.address }}"
+consul_nodename: "{{ ansible_nodename }}"
+consul_server: false
+consul_ui: false
+consul_backup: false
+
+consul_bin_dir: "/usr/local/bin"
+consul_config_dir: "/etc/consul"
+consul_data_dir: "/var/lib/consul"
+consul_openstack_data_device: "/dev/sdb1"
+
+consul_log_level: INFO
+consul_enable_syslog: true
+
+consul_encrypt: ""
+
+# Example:
+#
+# consul_servers: [nodeA.domain.com, nodeB.domain.com, nodeC.domain.com]
+consul_servers: []
+consul_rejoin_after_leave: true
+consul_enable_script_checks: true
```

Comment:
Any particular reason for the last two lines?  I wouldn't expect that we ever tell a Consul agent to leave the cluster except when shutting a VM down, in which case rejoining isn't needed.  And we are not planning to use Consul health checks in the first iteration, so we could leave health check scripts disabled for now.

Reply:
I noticed that Consul leaves the cluster automatically when shutting it down with SIGINT, so setting `rejoin_after_leave` seems indeed required, and my question is only about the health check scripts.

Reply:
Since I added the health checks anyway, I had to enable this (it's off by default).

Reply:
My understanding of the documentation is that this flag is only needed for script checks, e.g. the first one in https://www.consul.io/docs/agent/checks.html#check-definition.  I don't think you defined any script checks, but I may have missed them.

Reply:
Ah you're right. Originally, I did use script checks, in particular the sanity check script, but it was having permission issues and I didn't want to continue with it. But at this point, we can return to that in a future iteration. I'll turn this variable to `false` as default for now.